students/Craig_Morton/lesson03/list_lab.py

```diff
@@ -0,0 +1,107 @@
+# ------------------------------------------------- #
+# Title: Lesson 3, pt 2/4, List Lab Exercise
+# Dev:   Craig Morton
+# Date:  8/19/2018
+# Change Log: CraigM, 8/19/2018, List Lab Exercise
+#  ------------------------------------------------ #
+
+# !/usr/bin/env python3
+
+# Start of series 1 ##########
+
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.  Display the list (plain old print() is fine
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruit_list, "\n")
+
+# Ask the user for another fruit and add it to the end of the list
+# Display the list
+user_input = input("Please add a fruit to your list: ")
+fruit_list.append(user_input.title())
+print(fruit_list, "\n")
+
+# Ask the user for a fruit index value and display the value as well as the corresponding fruit
+user_input = " "
+index_value = None
+while not user_input.isdigit():
+    user_input = input("Please enter the index value of the fruit you wish to display: ")
+    if user_input.isdigit():
+        index_value = int(user_input)
+        if index_value > len(fruit_list):
+            user_input = " "
+print(index_value, fruit_list[index_value - 1], "\n")
```

Comment:
Good idea to indent this so it's part of the function, or use a return statement.  Also, recommend adding a catch if user input is not a digit.